src/client/Microsoft.Identity.Client/Extensibility/AcquireTokenOnBehalfOfParameterBuilderExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Microsoft.Identity.Client.Extensibility
+{
+    /// <summary>
+    /// Extension methods for the <see cref="AcquireTokenOnBehalfOfParameterBuilder" />
+    /// </summary>
+    public static class AcquireTokenOnBehalfOfParameterBuilderExtensions
+    {
+        /// <summary>
+        /// Only affects <see cref="ILongRunningWebApi.InitiateLongRunningProcessInWebApi(IEnumerable{string}, string, ref string)"/>.
+        /// When enabled, mimics MSAL 4.50.0 and below behavior - checks in cache for cached tokens first, 
+        /// and if not found, then uses user assertion to request new tokens from AAD.
+        /// When disabled (default behavior), doesn't search in cache, but uses the user assertion to retrieve tokens from AAD.
```

Comment:
This does not match with what is mentioned above. See https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/4135/files#diff-8510905684539ef8e938512ce363f8a33ab5f441a268981b07638be16bd3385aR24